.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Release Checklist
+about: Create a ticket to track a release
+title: ''
+labels: release
+assignees: ''
+
+---
+
+**Release**
+The release version and a description of the planned changes to be included in the release.
+
+**Issues**
+Link the major issues planned to be included in the release.
+
+**Documentation**
+Link the relevant documentation PRs for this release.
+
+**Checklist**
+- [ ] Update version in scripts/versions.sh and plugin/evm/version.go
+- [ ] Bump AvalancheGo dependency for RPCChainVM Compatibility
+- [ ] Add new entry in compatibility.json for RPCChainVM Compatibility
+- [ ] Update AvalancheGo compatibility in README
+- [ ] Bump cmd/simulator go mod (if needed)
+- [ ] Confirm release binaries are generated correctly
```

Comment:
how should this be done? (the last step)

Reply:
Check the release page to confirm that they are up after the goreleaser job has finished.

This may be a step to include a QA team to ensure that the whole release works on different OSs with the latest avalanchego and avalanche-cli releases as well. cc @rob-ava @yulin-dong @dasconnor .

When we had this issue in the past, it was reported by a user of the avalanche-cli.